### DISCUSSION TITLE
Criar mensagens para um comprador

### DISCUSSION THREAD

**Discussion by AlexdeJesusFS** [rodurma/forum-meli#14] [Geral]
Olá, estou tentando criar mensagens para um comprador como mostra a [documentação](https://developers.mercadolivre.com.br/pt_br/mensagens-post-venda), porém não consegui encontrar todos os parâmetros necessários. Não entendi o que é e onde posso encontrar o postman-token e o application_id, procurei por eles na documentação, mas não encontrei algo sobre.
Alguém sabe como enviar mensagens na API do mercado livre?

**Comment by Vitorkgb** [comment]
me manda um email que acho que consigo te explicar, vitorkgb@hotmail.com
